discojs/discojs-node/src/data/image_loader.spec.ts

```diff
@@ -16,24 +16,38 @@ const DIRS = {
 }
 
 const cifar10Mock: Task = {
-  taskID: 'cifar10',
+  id: 'cifar10',
   displayInformation: {},
   trainingInformation: {
+    modelID: 'cifar10',
+    epochs: 1,
+    roundDuration: 1,
+    batchSize: 1,
+    dataType: 'tabular',
```

Comment:
CIFAR10's `dataType` is probably 'image' rather than 'tabular'

Reply:
indeed, good catch!